Lib/inspect.py

```diff
@@ -819,7 +819,9 @@ def getfile(object):
     if ismodule(object):
         if getattr(object, '__file__', None):
             return object.__file__
-        raise TypeError('{!r} is a built-in module'.format(object))
+        if getattr(object, '__spec__', None) is not None:
+            raise TypeError(f'{object!r} is a built-in module')
+        raise TypeError(f'Cannot get source from {object!r}')
```

Comment:
```suggestion
        raise TypeError(f'cannot get source from {object!r}')
```

we don't capitalize error messages in this function

Reply:
Don't forget to update the test.